src/styles/blocks/calendar.scss

```
@@ -0,0 +1,49 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-between-item;
+  width: $width-calendar-body;
+  padding: $padding-calendar-body;
+
+  &__day {
+    position: relative;
+    width: $size-calendar-item;
+    height: $size-calendar-item;
+    background-color: $main-color-calendar-item;
+    border: solid 1px black;
+    box-sizing: border-box;
+    align-items: center;
+    justify-content: center;
+    display: flex;
+
+    &:hover {
+      cursor: pointer;
+      background-color: $hover-color-calendar-item;
+      transform: translateY(-20px);
+      transition-duration: 0.5s;
+    }
+  }
+
+  &__day::before {
+    content: '';
+    position: absolute;
+  }
+
+  @for $num from 1 through 31 {
+    &__day:nth-child(#{$num})::before {
+      content: '#{$num}';
+    }
+  }
+
+  @each $start-day, $margin-left in $days-of-week {
+    &--start-day-#{$start-day} :first-child {
+      margin-left: $margin-left;
```

```suggestion
      margin-left: $total-item-size * $margin-left;
```
and calculate it only one time here